### DISCUSSION TITLE
Google Search plugin

### DISCUSSION THREAD

**Discussion by ghost** [matamorphosis/Scrummage#35] [Q&A] [answered]
So I've provided the cx and API key for Google search and have the application and version name as shown
![Screenshot from 2021-11-03 12-47-53](https://user-images.githubusercontent.com/48868493/140062685-9011a476-d80e-423b-bc00-7bb7197484f6.png)

I'm getting no results for any query made.

In the terminal I'm getting the following when I run a Google search task
![Screenshot from 2021-11-03 12-49-30](https://user-images.githubusercontent.com/48868493/140062967-76f221eb-48f6-4e2b-a4b0-bd749f6c4102.png)

Any advice on why it's not working would be appreciated

**Comment by matamorphosis** [comment]
Hi @scars321, I saw you raised an issue but closed it off, were you able to resolve this?

**Comment by ghost** [comment]
Hi yes sorry it was just my issue that I've got listed here but closed it as I thought no need for it in both places

**Comment by ghost** [comment]
but no I've still not got it working @matamorphosis

**Comment by matamorphosis** [comment, marked as answer]
@scars321 no worries, someone else raised an issue for it where I have confirmed there was no issue preventing this from working. There was another, separate issue related to generating results, but this would not have caused the error described.

[Issue Here](https://github.com/matamorphosis/Scrummage/issues/37)

One other thing, is that the value of the "application_version" field that I use is "v1", not 1.0.

**Reply by ghost** [reply]
v1......... Oh dear I need to wake up haha, thanks dude that's working.

**Reply by matamorphosis** [reply]
@h0bit1 no worries at all, some of the nuances with these third-party APIs can mess with anyone's head, hopefully over time these discussions will start to make it easier :)
